src/arrayMethodJoin.js

```diff
@@ -4,8 +4,20 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let str = '';
```

Comment:
dnt tr t mk r cd shrtr b rmvng vwls. But if I wanted you to understand what I wrote, I would write: "don't try to make your code shorter by removing vowels "